routes/api/courses.js

```diff
@@ -74,6 +75,81 @@ router.delete("/:courseId", passport.authenticate('jwt', {session: false}), asyn
       return res.status(200).json({ status: "success" })
     })
     .catch(() => res.status(404).json({ status: "failure", error: "course not found" }))
-})
+});
+
+// Add a student to a course with student ID
+router.put("/:courseId/:studentId", passport.authenticate('jwt', {session: false}), async (req, res) => {
```

Comment:
Unless you can only add one student at a time on the frontend, it would be better to pass an array of studentIds in the request body and add all of those studentIds to the course's student's array.